<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<ows:ExceptionReport version="2.0.0"
    xsd:schemaLocation="http://www.opengis.net/ows/2.0 http://schemas.opengis.net/ows/2.0/owsExceptionReport.xsd"
      >
    <ows:Exception exceptionCode="RuntimeError">
        <ows:ExceptionText>Runtime error while processing request</ows:ExceptionText>
    </ows:Exception>
    <ows:Exception exceptionCode="RuntimeError">
        <ows:ExceptionText>petascope.wcps2.error.managed.processing.OutOfBoundsSubsettingException : Invalid subsetting: (1950-01-03T13:05:00:1950-01-08T13:05:00) is not within coverage's domain (1950-01-01T00:00:00.000Z:1950-01-07T00:00:00.000Z) for axis t.</ows:ExceptionText>
    </ows:Exception>

</ows:ExceptionReport>

